src/api/app/models/user.rb

```diff
@@ -882,7 +872,7 @@ def hash_string(value)
                     'sha256crypt' => 5 }
     if deprecated_password_hash_type == 'md5'
       Digest::MD5.hexdigest(value + deprecated_password_salt)
-    elsif crypt2index.keys.include?(deprecated_password_hash_type)
+    elsif crypt2index.key?(deprecated_password_hash_type)
```

Comment:
What does this have to do with attributes?

Reply:
atom beautifies the code on save

Reply:
Then atom should also make it an extra commit and describe why...

Reply:
Just as my cat owning neighbour said: I will talk to it, but I'm not sure it will change :)